src/test/datascience/notebook/helper.ts

```diff
@@ -736,7 +736,9 @@ export async function waitForTextOutput(
         async () => assertHasTextOutputInVSCode(cell, text, index, isExactMatch),
         timeout,
         () =>
-            `Output does not contain provided text '${text}' for Cell ${cell.index + 1}, it is ${getCellOutputs(cell)}`
+            `A after ${timeout}ms output does not contain provided text '${text}' for Cell ${
```

Comment:
Looks like a findreplace mistake?

Reply:
No it was deliberate, i wanted to include the timeout in the error message. Noticed this test was failing on CI